pyproject.toml

```diff
@@ -73,13 +73,15 @@ build = [
 
 [tool.cibuildwheel.linux]
 archs = ["x86_64", "aarch64"]
-before-all = "make c_lib"
 
 [tool.cibuildwheel.macos]
 archs = ["x86_64", "universal2", "arm64"]
 environment = { MACOSX_DEPLOYMENT_TARGET="10.15" }
-before-build = "make clean && make c_lib"
+before-build = "make clean"
 
 [tool.cibuildwheel.windows]
 archs = ["AMD64"]
-before-build = '"C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC\\Auxiliary\\Build\\vcvars64.bat" && cl "ada_url\\ada.cpp" /c /nologo /Fo"ada_url\\ada.o" /O2 /GL /MD /W3 /EHsc /std:c++17'
+
+[[tool.cibuildwheel.overrides]]
+select = "pp39-win_amd64"
```

Comment:
Could you leave a comment about why this one is special?

Reply:
Added a comment with link to https://github.com/pypy/pypy/issues/5027